### PR TITLE
Update dependencies

### LINKED ISSUE #36: Cannot autowire service "Prooph\ProophessorDo\Model\User\Service\ChecksUniqueUsersEmailAddress"
(1/1) RuntimeException
Cannot autowire service "Prooph\ProophessorDo\Model\User\Service\ChecksUniqueUsersEmailAddress": argument "$userFinder" of method "Prooph\ProophessorDo\Infrastructure\Service\ChecksUniqueUsersEmailAddressFromReadModel::__construct()" references class "Prooph\ProophessorDo\Projection\User\UserFinder" but no such service exists. You should maybe alias this class to the existing "proophdo.todo_projection.user_finder" service.
--

### PR DESCRIPTION
**Pull request by codeliner**
Fixes #36

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -94,7 +94,8 @@
         "auto-scripts": {
             "make cache-warmup": "script",
             "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
```
Comment (dave-redfern) [inline composer.json:96]: Did you mean to leave in the 2 preceding commands?

Right now, the cache is warmed and then immediately cleared. While the second `assets:install` without symlink and relative will copy the assets into the already symlinked locations and that might have unintended side-effects.

Comment (codeliner) [reply composer.json:96]: Can you send a PR to correct this?

Comment (dave-redfern) [reply composer.json:96]: @codeliner Done in PR #39

Comment (codeliner) [reply composer.json:96]: cool, thx!

**Pull request merged**